.pre-commit-config.yaml

```diff
@@ -1,28 +1,28 @@
 exclude: ^tests/data/
 repos:
-- repo: https://gitlab.com/pycqa/flake8
```

Comment:
No need to modify this file.

Reply:
Sometimes this path cannot be correctly processed by pre-commit.